_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
 ---
 layout: post
-title: "NewPipe 0.20.0 released: A new major version, a new unified player, a new notification experience... A lot is new this time!"
+title: "NewPipe 0.20.0 released: Major version, new unified player, new notifications. More things."
 short: "NewPipe 0.20.0 released"
 date: 2020-10-04 05:00:00 +0100
 categories: [pinned, release]
-author: Opusforlife & Poolitzer
+author: Opusforlife, Poolitzer and Allan Nordhøy
 image: newpipe
 excerpt_separator: <!-- more -->
 ---
 
-Hey there! Guess who is back? We are. Admittedly, we're quite a bit late with this release. But there is a _very_ good reason for that. Are you ready for it? Sit down, grab some popcorn and a beverage of your choice. This is going to be long.
```

Comment:
This is bait, and fluff at that. Pontificating how long something is doesn't help. The reader is not enticed, the readership is lost.

Reply:
It's filler text. There is a major subheading just after this, so it doesn't matter if the reader really reads this part or not. I expect most of them will skip to the subheading, anyway.

Reply:
It continues in the lead paragraph. Writing things the reader isn't expected to read, to start? ???

Reply:
It's a release blog post. We write stuff we want to talk about in this release and the reader reads it. Are there any rules about what we can talk about and what not?